consensus/polybft/statesyncrelayer/state_sync_relayer.go

```diff
@@ -151,7 +163,7 @@ func (r *StateSyncRelayer) queryStateSyncProof(stateSyncID string) (*types.State
 		return nil, err
 	}
 
-	r.logger.Debug(fmt.Sprintf("state sync proof: %v", stateSyncProof))
+	r.logger.Info(fmt.Sprintf("state sync proof: %v", stateSyncProof))
```

Comment:
Not sure about this, it can spam logs quite a bit if there are going to be a lot of state syncs, so maybe leave it on debug log level? WDYT?

Reply:
I think debug is fine